backend/cn/main.ml

```diff
@@ -421,90 +527,164 @@ module Executable_spec_flags = struct
 let output_decorated_dir =
   let doc = "output a version of the translation unit decorated with C runtime
   translations of the CN annotations to the provided directory" in
-  Arg.(value & opt (some string) None & info ["output_decorated_dir"] ~docv:"FILE" ~doc)
+  Arg.(value & opt (some string) None & info ["output-decorated-dir"] ~docv:"FILE" ~doc)
 
 let output_decorated =
   let doc = "output a version of the translation unit decorated with C runtime
   translations of the CN annotations." in
-  Arg.(value & opt (some string) None & info ["output_decorated"] ~docv:"FILE" ~doc)
+  Arg.(value & opt (some string) None & info ["output-decorated"] ~docv:"FILE" ~doc)
 
 let with_ownership_checking =
   let doc = "Enable ownership checking within CN runtime testing" in
-  Arg.(value & flag & info ["with_ownership_checking"] ~doc)
+  Arg.(value & flag & info ["with-ownership-checking"] ~doc)
 
 let copy_source_dir =
   let doc = "Copy non-CN annotated files into output_decorated_dir for CN runtime testing" in
-  Arg.(value & flag & info ["copy_source_dir"] ~doc)
+  Arg.(value & flag & info ["copy-source-dir"] ~doc)
 
 end
 
-module Lemma = struct
+module Lemma_flags = struct
 let lemmata =
   let doc = "lemmata generation mode (target filename)" in
   Arg.(value & opt (some string) None & info ["lemmata"] ~docv:"FILE" ~doc)
 
 end
 
-module Test_Generation_Flags = struct
+let check_cmd =
+  let open Term in
+  let check_t =
+    const check $
+    Common_flags.file $
+    Common_flags.macros $
+    Common_flags.incl_dirs $
+    Common_flags.incl_files $
+    Verify_flags.json $
+    Verify_flags.state_file $
+    Verify_flags.output_dir $
+    Common_flags.csv_times $
+    Common_flags.log_times $
+    Common_flags.astprints $
+    Common_flags.use_peval $
+    Common_flags.no_inherit_loc $
+    Common_flags.magic_comment_char_dollar
+  in
+  let doc = "Run CN well-formedness check" in
+  let info = Cmd.info "check" ~doc in
```

Comment:
Can you call it `lint-spec`? Just because the distinction between `verify` and `check` may be confusing (we can always change it later, but for now).

Reply:
What about `cn sanity-check`?

Reply:
I like the last one

Reply:
It's also my favorite, so I changed `check` to `wf`. It being opaque stops users from assuming its functionality.